packages/components/addon/components/hds/tabs/index.js

```diff
@@ -79,6 +81,21 @@ export default class HdsTabsIndexComponent extends Component {
     this.panelNodes[tabIndex].focus();
   }
 
+  setTabIndicator(tabIndex) {
+    const tabElem = this.tabNodes[tabIndex];
+    const tabsParentElem = tabElem.closest('.hds-tabs');
+
+    const tabLeftPos = tabElem.parentNode.offsetLeft;
+    const tabWidth = tabElem.parentNode.offsetWidth;
+
+    // Set CSS custom properties for indicator
+    tabsParentElem.style.setProperty(
+      '--indicatior-left-pos',
+      tabLeftPos + 'px'
+    );
+    tabsParentElem.style.setProperty('--indicatior-width', tabWidth + 'px');
+  }
```

Comment:
Let me know if this violates any Ember best practices and/or if there's better ways to do this.